Loger/FileService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    public class FileService
+    {
+        public void SaveFile(string[] message)
+        {
+            this.CreateFolder();
+            int count = new Random().Next(1, 100);
+            int countFile = Directory.GetFiles("Log").Length;
```

Comment:
use can combine in single line 
var files = Directory.GetFiles("Log");
string[] countFile = files.Length;